scriptworker/cot/verify.py

```diff
@@ -795,6 +795,10 @@ def get_all_artifacts_per_task_id(chain, upstream_artifacts):
                 dict_=all_artifacts_per_task_id, key=upstream_dict['taskId'], item=upstream_dict['paths']
```

Comment:
Nit: Add `sorted in the pydoc. For instance: 

```py
"""
    Returns:
        dict: list of sorted paths to downloaded artifacts ordered by taskId
"""
```

Reply:
Done.